docs/getting-started/quickstart.md

```diff
@@ -8,13 +8,14 @@ import TabItem from '@theme/TabItem';
 It's easy to get started with Onepanel. First, you install the CLI (`opctl`) and then using `opctl`, you generate a `params.yaml` file and update it to configure your deployment. Once complete, you can access your deployment from any browser using your Kubernetes authentication token. Finally, you can run a Workflow or create a Workspace.
 
 :::important
-The steps in the Quick start allow you to quickly setup a Onepanel cluster for testing. To setup a production cluster with TLS and auto scaling enabled see [instructions for your cloud provider](/docs/deployment/overview#installing-on-public-cloud)
+The steps in the Quick start allow you to quickly setup a Onepanel cluster for testing. To setup a production cluster with TLS and auto scaling enabled see [instructions for your cloud provider](/docs/deployment/overview#installing-on-public-cloud).
+
+You can also [add components](/docs/deployment/upgrade/overview) at later time to make this cluster production ready.
 :::
 
-## Step 0: Understand the concepts
-First, take a look at [concepts](/docs/getting-started/concepts/namespaces) to understand the different components in Onepanel.
+Before getting started, take a look at [concepts](/docs/getting-started/concepts/namespaces) to understand the different components in Onepanel.
 
-## Step 1: Create a Kubernetes cluster
+## Step 0: Create a Kubernetes cluster
```

Comment:
@rushtehrani What about including the way to set the azure cluster kubernetes version?
```
 --kubernetes-version 1.12.6
```

Reply:
Trying to keep the instructions minimal for the quick start, but adding it to the AKS deployment instructions makes perfect sense which I see you've already done.

Reply:
Minimal makes sense. Not a blocker by any means.